src/components/Categories.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <h2>Under construction</h2>;
+
+export default Categories;
```

Comment:
- [ ] Please follows the ducks pattern, `Catergories.js` should be in a folder called `categories`

Reply:
Hello @AshakaE,
I followed this pattern(the duck pattern) for the redux files. Is it necessary to put each component in separate files?
/src
    |---/components
    |---/redux
        |--- /books
                | books.js
        |--- /categories
                | categories.js
        | configureStore.js
    | index.js